package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-azure-auth",
-  "version": "1.8.18",
+  "version": "1.8.19",
```

Comment:
please leave the version update to lib publishing process.  just remove it from PR pls